app/src/org/commcare/network/CommcareRequestGenerator.java

```diff
@@ -226,17 +242,19 @@ private static String getDigest() {
 
     @Override
     public Response<ResponseBody> simpleGet(String uri) throws IOException {
-        return simpleGet(uri, new HashMap());
+        return simpleGet(uri, new HashMap<>(), new HashMap<>());
     }
 
     @Override
-    public Response<ResponseBody> simpleGet(String uri, Map<String, String> httpParams) throws IOException {
+    public Response<ResponseBody> simpleGet(String uri, Map<String, String> httpParams, Map<String, String> httpHeaders) throws IOException {
+        HashMap<String, String> headers = new HashMap<>(getHeaders(null));

```

Comment:
Earlier, we were using `getHeaders("")` and now we're using `getHeaders(null)` which means now we'd not be having `X_COMMCAREHQ_LAST_SYNC_TOKEN` in the headers. Is this expected?

Reply:
yes, I don't think it serves any purpose to have `X_COMMCAREHQ_LAST_SYNC_TOKEN` as empty in headers.